src/Sentry/Protocol/Response.cs

```diff
@@ -144,7 +147,7 @@ public static Response FromJson(JsonElement json)
             BodySize = bodySize,
             Cookies = cookies,
             Data = data,
-            InternalHeaders = headers?.WhereNotNullValue().ToDict(),
+            InternalHeaders = headers.Redact(),
```

Comment:
With the implicit operator, do we need to call the method here?

Reply:
We do unfortunately. InternalHeaders is nullable and that can't be done with an implicit operator.